src/utils/wdyr.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+if (process.env.NODE_ENV === 'development') {
+  // eslint-disable-next-line global-require
+  const whyDidYouRender = require('@welldone-software/why-did-you-render');
+  whyDidYouRender(React, {
```

Comment:
what is this?

Reply:
It's a library that we can use to debug rerendering problems : https://github.com/welldone-software/why-did-you-render